provider/runbook_resource_test.go

```diff
@@ -296,6 +316,10 @@ func testAccCheckRunbookResourceExistsWithAttributes_update(resourceName string)
 			return fmt.Errorf("Unexpected description. Expected: %s, got: %s", expected, got)
 		}
 
+		if runbookResponse.AttachmentRule == nil {
+			return fmt.Errorf("Unexpected attachment_rule. Expected: %s, got: %s", expected, got)
+		}
+
```

Comment:
do you think we could do a more in-depth check here? something kinda like we do for config, like this:
```suggestion
		if runbookResponse.AttachmentRule == nil {
			return fmt.Errorf("Unexpected attachment_rule. Expected: %s, got: %s", expected, got)
		}
		var attachmentRule []byte
		if len(runbookResponse.AttachmentRule) > 0 {
			attachmentRule, err = json.Marshal(runbookResponse.AttachmentRule)
			if err != nil {
				return fmt.Errorf("Unexpected error converting attachment_rule to JSON: %v", err)
			}
		}
		if runbookResource.Primary.Attributes["attachment_rule"] != string(attachmentRule) {
			return fmt.Errorf("Unexpected attachment_rule. Expected %s, got: %s", runbookResponse.AttachmentRule, runbookResource.Primary.Attributes["attachment_rule"])
		}

```